### PR TITLE
export for cdo and bureau AB

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1370)

Going to work on the other two in another PR after updating the payload.
~From Chris' email(will update export):~
- [ ] ~All languages for the person (including scores)~
- [ ] ~City, Country (for overseas)  -- Bureau/Officer (for domestic DC positions) – City, State (for other domestic positions)~
- [x] All of their skills.

Woo Pydash!

### CODE REVIEW

File: talentmap_api/cdo/services/available_bidders.py [hunk 1]
```diff
@@ -70,25 +71,35 @@ def get_available_bidders_csv(jwt_token):
...
-            smart_str("=\"%s\"" % record["oc_reason"]),
+            smart_str(', '.join(skills)),
+            smart_str("=\"%s\"" % record["grade"]),
+            smart_str(ted),
+            smart_str(record["post"]["location"]["country"]),
             smart_str("=\"%s\"" % record["oc_bureau"]),
+            smart_str("=\"%s\"" % record["oc_reason"]),
             smart_str("=\"%s\"" % record["comments"]),
             smart_str("=\"%s\"" % record["is_shared"]),
         ])
```
Comment (mjoyce91) [inline talentmap_api/cdo/services/available_bidders.py:105]: Should we make empty strings say "None listed" so that it matches the FE?

Comment (elizabeth-jimenez) [reply talentmap_api/cdo/services/available_bidders.py:105]: 💯  good point; i'll update

File: talentmap_api/cdo/services/available_bidders.py [hunk 2]
```diff
@@ -78,29 +78,59 @@ def get_available_bidders_csv(jwt_token):
...
+                if not skills:
+                    fields[f] = "No Skills listed"
+                else:
+                    fields[f] = ', '.join(skills)
+            else:
+                try:
+                    if f is "ted":
+                        fields[f] = maya.parse(record["TED"]).datetime().strftime('%m/%d/%Y')
+                    elif f is "post":
+                        fields[f] = record["post"]["location"]["country"]
```
Comment (elizabeth-jimenez) [inline talentmap_api/cdo/services/available_bidders.py:114]: Im going to play around with this and see if i can pass in data paths. If I can, i can abstract this to a common function

File: talentmap_api/common/common_helpers.py [hunk 3]
```diff
@@ -441,3 +442,28 @@ def validate_values(query_val, accepted_values):
     validated_list = ",".join(map(str, validated_list))
 
     return validated_list
+
+def formatCSV(data, fieldsInfo):
```
Comment (mjoyce91) [inline talentmap_api/common/common_helpers.py:446]: I like the use of the `fieldsInfo` object

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**